API.Tests/Services/DirectoryServiceTests.cs

```diff
@@ -505,7 +505,7 @@ public void CopyFilesToDirectory_ShouldMoveAllFiles()
             }
 
             var ds = new DirectoryService(Substitute.For<ILogger<DirectoryService>>(), fileSystem);
-            ds.CopyFilesToDirectory(new []{$"{testDirectory}file_{0}.zip", $"{testDirectory}file_{1}.zip"}, "/manga/output/");
+            ds.CopyFilesToDirectory(new[] { $"{testDirectory}file_{0}.zip", $"{testDirectory}file_{1}.zip" }, "/manga/output/");
```

Comment:
Please leave the space between new and [].

Reply:
I did not actually do that. VS did automatically. I'll try to reverse it though